Makefile

```diff
@@ -746,6 +746,11 @@ docs:
 run-docs:
 	$(MAKE) -C docs run
 
+.PHONY: docs-lint
+docs-lint:
+	$(MAKE) -C docs lint
+
+
```

Comment:
extra whitespace

Reply:
I have to ask - is this a question or statement? Why not remove it?

Reply:
I will remove it.  I only saw this issue when I was putting the PR up at the end of the day.

Reply:
removed in b1a2ffd